test/recipes/81-test_cmp_cli.t

```diff
@@ -222,7 +221,7 @@ indir data_dir() => sub {
                 test_cmp_cli_aspect($server_name, $aspect, $tests);
             };
         };
-        stop_mock_server() if $launch_mock;
+        stop_mock_server($pid) if $pid;
```

Comment:
Isn't the value of `$pid` undefined if `$server_name` isn't "Mock"?

Reply:
> Isn't the value of `$pid` undefined if `$server_name` isn't "Mock"?

Yes (because it is a scalar variable that has not been initialized in this case), 
and then it is considered 'false' in the `if` context - 
I've just confirmed the latter for myself at <https://perlmaven.com/boolean-values-in-perl>.

Reply:
Ah - ok fair enough.